modules/5_install/install.tf

```diff
@@ -448,3 +448,13 @@ resource "null_resource" "csi_driver_config" {
     ]
   }
 }
+
+resource "ibm_pi_operations" "fips_bastion_reboot" {
```

Comment:
@yussufsh 
 We have tested reboot resource in 1_prepare module, reboot working fine but whenever next resource trying to do ssh to bastion node it was giving us error, so we have added this code at the end of install module after all deployment steps are done, and also it will work only if FIPS set to true.

Reply:
We have tested that after deployment which includes reboot we are able to login to cluster.